install.md

```diff
@@ -118,42 +116,134 @@ yum install -y \
 Debian, Ubuntu, and related distributions:
 
 ```bash
-apt-get install -y \
+sudo apt-get install \
   btrfs-tools \
   git \
   golang-go \
   go-md2man \
   iptables \
   libassuan-dev \
+  libc6-dev \
   libdevmapper-dev \
   libglib2.0-dev \
-  libc6-dev \
-  libgpgme11-dev \
+  libgpgme-dev \
   libgpg-error-dev \
+  libostree-dev \
   libprotobuf-dev \
   libprotobuf-c0-dev \
   libseccomp-dev \
   libselinux1-dev \
+  libsystemd-dev \
   pkg-config \
+  runc \
   uidmap
 ```
 
-Debian, Ubuntu, and related distributions will also need to do the following setup:
+### Building missing dependencies
+
+If any dependencies cannot be installed or are not sufficiently current, they have to be built from source.
+This will mainly affect Debian, Ubuntu, and related distributions, or RHEL where no subscription is active (e.g. Cloud VMs).
+
+#### ostree
+
+A copy of the development libraries for `ostree` is necessary, either in the form of the `libostree-dev` package
+from the [flatpak](https://launchpad.net/~alexlarsson/+archive/ubuntu/flatpak) PPA,
+or built [from source](https://github.com/ostreedev/ostree/blob/master/docs/contributing-tutorial.md)
+(see also [here](https://ostree.readthedocs.io/en/latest/#building)). As of Ubuntu 18.04, `libostree-dev` is available in the main repositories,
+and the PPA is no longer required.
+
+To build, use the following (running `make` can take a while):
+```bash
+git clone https://github.com/ostreedev/ostree ~/ostree
+cd ~/ostree
+git submodule update --init
```

Comment:
This section is mainly built on https://github.com/ostreedev/ostree/blob/master/docs/contributing-tutorial.md, but enhanced with package dependencies that are necessary for building, a solution for ostreedev/ostree#1374, and setting the library locations in a way that the linker has a better chance of finding them (this is relevant when trying to build podman with an `selinux` buildtag)